pkg/system/phase4_configuring.go

```diff
@@ -32,6 +32,13 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 )
 
+const (
+	ibmEndpoint         = "https://s3.direct.%s.cloud-object-storage.appdomain.cloud"
+	ibmLocation         = "%s-standard"
+	ibmCOSCred          = "ibm-cloud-cos-creds"
+	ibmCOSCredDefaultNS = "openshift-storage"
```

Comment:
I think we should avoid defining a default namespace for these credentials.
It should always be loaded from the namespace of the NooBaa system so that it is consistent with the RBAC permissions we require.
For example - in case we deploy noobaa on another namespace, why should it default to trying openshift-storage namespace for credentials lookup - these credentials were not given to it, and it shouldn't access it.
My suggestion is that the operator will lookup this fixed name secret in the noobaa system namespace.

Reply:
Removed the default namespace for cred secrets